catalyst-gateway/bin/src/db/index/block/txo/insert_txo_asset.rs

```diff
@@ -17,14 +21,14 @@ const INSERT_TXO_ASSET_QUERY: &str = include_str!("./cql/insert_txo_asset.cql");
 /// (Superset of data to support both Staked and Unstaked TXO records.)
 #[derive(SerializeRow, Debug)]
 pub(crate) struct Params {
-    /// Stake Address - Binary 28 bytes. 0 bytes = not staked.
-    stake_address: Vec<u8>,
+    /// Stake Address - Binary 29 bytes. 0 bytes = not staked.
+    stake_hash: Vec<u8>,
```

Comment:
```suggestion
    stake_address: Vec<u8>,
```

Never use 0 bytes, if it has no stake address, don't store it here.